### PR TITLE
Initial {deploy,destroy}-demo-env refactor

### PR DESCRIPTION
**Pull request by sodar**

### CODE REVIEW

**Comment by sodar** [comment]
To reviewers: commit `3f38c5e9c04e1c680ce13451f0b87b3a46221c3f` only adds original files from internal repository

File: ansible/roles/vmware-vm/tasks/destroy.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+- name: Destroy VM '{{ inventory_hostname }}'
+  delegate_to: localhost
+  vmware_guest:
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_user }}"
+    password: "{{ vcenter_password }}"
+    validate_certs: "{{ validate_certs }}"
+
+    folder: "{{ datacenter_name }}/vm/{{ testenv_folder }}"
```
Comment (kaweue) [inline ansible/roles/vmware-vm/tasks/destroy.yml:11]: Is this required (folder) ?

File: jenkinsfiles/Jenkinsfile.destroy-demo-env [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            steps {
+                dir("ansible") {
+                    ansiblePlaybook extras: "-e @vmware-vm.vars",
+                                    inventory: "inventory.testenv",
+                                    playbook: "vmware-destroy-testenv.yml"
+                }
+            }
+        }
+    }
+}
```
Comment (kaweue) [inline jenkinsfiles/Jenkinsfile.destroy-demo-env]: Missing line

**Review by kaweue — CHANGES_REQUESTED** [review]

File: ansible/roles/vmware-vm/tasks/destroy.yml (hunk 1 above)
Comment (sodar) [reply ansible/roles/vmware-vm/tasks/destroy.yml:11]: Yes, specifying a folder is required

**Review by kaweue — APPROVED** [review]

**Pull request merged**